goldenmaster/apigear/mqtt/private/mqttcwrapper.cpp

```diff
@@ -147,18 +147,25 @@ CWrapper::CWrapper(const std::string& clientID)
 
 CWrapper::~CWrapper() = default;
 
-int CWrapper::subscribeToConnectionStatus(OnConnectionStatusChangedCallBackFunction callBack)
+int CWrapper::createRandomConnectionStatusId()
```

Comment:
I think that "unique" is more important than random, I'd name it 
getUniqueConnectionStatusId

Reply:
minor, but ok

Reply:
done